src/kernel.ts

```diff
@@ -547,25 +576,6 @@ class Kernel {
     };
   }
 
-  /**
-   * Close the Websocket.
-   */
-  private _stopChannels(): void {
```

Comment:
I don't want to tie it to a websocket per se, since that it an implementation detail.